ideascube/library/views.py

```diff
@@ -134,7 +134,8 @@ class SpecimenCreate(CreateView):
     def get_initial(self):
         book = get_object_or_404(Book, pk=self.kwargs['book_pk'])
 
-        return {'book': book, 'location': get_server_name()}
+        return {
+            'book': book, 'location': Configuration('server', 'site-name')}
```

Comment:
I dislike abbreviations for API names.

Having to introduce a newline is the perfectly normal thing to do in this specific case IMHO.

And in general, I'd rather have a discussion about raising the limit to (for example) 120 before I'd consider writing poor API.